[
  {
    "id": "https://github.com/microsoft/VPTQ/issues/119",
    "source": {
      "issue_number": 119
    },
    "initial_question": {
      "title": "When I use the parameter npercent=1 to quantize the model, I have the following problem\uff1a",
      "body": "/VPTQ/vptq/vptq.py\", line 443, in get_error\n    weight = weight.reshape(qweight.shape)\nRuntimeError: shape '[3584, 3548]' is invalid for input of size 12845056\n\n"
    },
    "satisfaction_conditions": [
      "Vector length parameters must be properly configured for the outlier quantization",
      "Quantization configuration must properly handle outlier allocation"
    ],
    "created_at": "2024-11-15T14:35:02Z"
  }
]